### PR TITLE
Rename "IdData" class to "SavedIdData"

### PR DESCRIPTION
**Pull request by robflop**
As discussed in #107. I focused only on direct references to the class, without changing variable/event names.

If there are further instances that I missed, please let know and I will also change them.

### CODE REVIEW

**Comment by baltpeter** [comment]
Deploy preview for *datenanfragen* ready!

Built with commit bb9361a7f4d3ec54d8f959bfc60f64088c1213db

https://deploy-preview-109--datenanfragen.netlify.com

File: src/Components/RequestGeneratorBuilder.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default class RequestGeneratorBuilder extends preact.Component {
             });
         });
 
-        if (Privacy.isAllowed(PRIVACY_ACTIONS.SAVE_ID_DATA)) this.idData = new IdData();
+        if (Privacy.isAllowed(PRIVACY_ACTIONS.SAVE_ID_DATA)) this.idData = new SavedIdData();
```
Comment (zner0L) [inline src/Components/RequestGeneratorBuilder.js]: Ok, so now that I look at the code occurrences of the class again I can see that I missed some things. All variables that contain instances of the class should of course be renamed as well. The confusion is real. So rename this as well please.

Comment (zner0L) [reply src/Components/RequestGeneratorBuilder.js]: Just for clarity: Occurences of `id_data` that do not contain an instance of `SavedIdData` should not be renamed, like this one: `this.state.request.id_data`.

Comment (robflop) [reply src/Components/RequestGeneratorBuilder.js]: No worries, i'll look into it.

**Comment by robflop** [comment]
I've looked through the files again and hope I caught all changes you were thinking of. Documentation (e.g. IdDataElement) and other class names (e.g. IdDataControls) left unchanged.

**Comment by robflop** [comment]
I've reversed the misnamed properties, apologies for that.

**Comment by robflop** [comment]
Will take a look, thank you.

**Pull request merged**